tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaRequestBuilder.cs

```diff
@@ -39,12 +40,14 @@ WebRequest ILambdaExtensionRequest.GetEndInvocationRequest(Scope scope, bool isE
             var request = WebRequest.Create(Uri + EndInvocationPath);
             request.Method = "POST";
             request.Headers.Set(HttpHeaderNames.TracingEnabled, "false");
-            if (scope != null)
+
+            if (scope is { Span: var span })
             {
-                request.Headers.Set(HttpHeaderNames.TraceId, scope.Span.TraceId.ToString());
-                request.Headers.Set(HttpHeaderNames.SpanId, scope.Span.SpanId.ToString());
+                // TODO: add support for 128-bit trace ids in serverless
```

Comment:
Do you know if that has been discussed with them? Should we do it now? Should we file a ticket not to forget?

Reply:
We're only adding 128-bit support to tracing for now. Other products will follow.

Serverless will need to support either the W3C `traceparent` header (which supports 128-bits trace ids) or the `x-datadog-tags` header with the higher-order 64 bits.